gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/process/traversal/translator/GolangTranslatorTest.java

```diff
@@ -0,0 +1,118 @@
+        final String gremlinAsGo = translator.translate(
+                g.V().has("person", "name", "marko").
+                        where(outE().count().is(2).and(__.not(inE().count().is(3)))).asAdmin().getBytecode()).getScript();
+        assertEquals("g.V().Has(\"person\", \"name\", \"marko\").Where(gremlingo.AnonTrav__.OutE().Count().Is(2).And(gremlingo.AnonTrav__.Not(gremlingo.AnonTrav__.InE().Count().Is(3))))", gremlinAsGo);
+    }
+
+    // TODO AN-987: TraversalStrategy implementation in Gremlin-go
+    @Ignore
+    @Test
+    public void shouldTranslateStrategies() {
+        assertEquals("g.withStrategies(*[TraversalStrategy('ReadOnlyStrategy', None, 'org.apache.tinkerpop.gremlin.process.traversal.strategy.verification.ReadOnlyStrategy'),TraversalStrategy('SubgraphStrategy',{'checkAdjacentVertices':False,'vertices':__.hasLabel('person')}, 'org.apache.tinkerpop.gremlin.process.traversal.strategy.decoration.SubgraphStrategy'),TraversalStrategy('SeedStrategy',{'seed':999999,'strategy':'org.apache.tinkerpop.gremlin.process.traversal.strategy.decoration.SeedStrategy'}, 'org.apache.tinkerpop.gremlin.process.traversal.strategy.decoration.SeedStrategy')]).V().has('name')",
+                translator.translate(g.withStrategies(ReadOnlyStrategy.instance(),
+                                SubgraphStrategy.build().checkAdjacentVertices(false).vertices(hasLabel("person")).create(),
+                                new SeedStrategy(999999)).
+                        V().has("name").asAdmin().getBytecode()).getScript());
+    }
+
+    // TODO: AN-1037 Lambda support in Gremlin-Go
+    @Ignore
+    @Test
+    public void shouldTranslateLambdas() {
+        final Bytecode bytecode = g.withSideEffect("lengthSum", 0).withSack(1)
+                .V()
+                .filter(Lambda.predicate("x -> x.get().label() == 'person'", "gremlin-groovy"))
+                .flatMap(Lambda.function("it.get().vertices(Direction.OUT)", "gremlin-groovy"))
+                .map(Lambda.<Traverser<Object>, Integer>function("x -> x : len(x.get().value('name'))", "gremlin-groovy"))
+                .sideEffect(Lambda.consumer("x -> x.sideEffects(\"lengthSum\", x.sideEffects('lengthSum') + x.get())    ", "gremlin-groovy"))
+                .order().by(Lambda.comparator("a,b -> a == b ? 0 : (a > b) ? 1 : -1)", "gremlin-groovy"))
+                .sack(Lambda.biFunction("a,b -> a + b", "gremlin-groovy"))
+                .asAdmin().getBytecode();
+        assertEquals("g.withSideEffect('lengthSum',0).withSack(1).V().filter(lambda: \"x -> x.get().label() == 'person'\").flatMap(lambda: \"it.get().vertices(Direction.OUT)\").map(lambda: \"x -> x : len(x.get().value('name'))\").sideEffect(lambda: \"x -> x.sideEffects(\\\"lengthSum\\\", x.sideEffects('lengthSum') + x.get())\").order().by(lambda: \"a,b -> a == b ? 0 : (a > b) ? 1 : -1)\").sack(lambda: \"a,b -> a + b\")",
+                translator.translate(bytecode).getScript());
+    }
+}
```

Comment:
Whats the coverage of these tests?

Reply:
Whatever the coverage of the Python tests are.